composer.json

```diff
@@ -0,0 +1,27 @@
+{
+	"name": "qa-tools/behat-example",
+	"license": "BSD-3-Clause",
+	"require": {
+		"php": ">=5.3.1",
+		"qa-tools/qa-tools": "~1.0@dev",
+		"qa-tools/behat-extension": "dev-cleanup",
+		"mindplay/annotations": "~1.2@dev",
+		"behat/mink": "~1.5@dev",
```

Comment:
It shouldn't be even in cause the all dependencies should be resolved through the extension itself.